plugin-enclave-gradle/src/main/kotlin/com/r3/conclave/plugin/enclave/gradle/LinuxExec.kt

```diff
@@ -115,21 +121,14 @@ open class LinuxExec @Inject constructor(objects: ObjectFactory) : ConclaveTask(
 
     /** Returns the output of the command executed in the container. */
     fun execWithOutput(params: List<String>): String {
-        val args: List<String> = listOf(
-            "docker",
-            "run",
-            "-i",
-            "--rm",
-            "-v",
-            "${baseDirectory.get()}:/project",
-            tag.get()
-        ) + params.map { it.replace(baseDirectory.get(), "/project").replace("\\", "/") }
+        val args: List<String> =
+            dockerRunArgs + params.map { it.replace(baseDirectory.get(), "/project").replace("\\", "/") }
```

Comment:
Again, more duplication. Instead of the lazy args list, have a function that creates the full args list by taking in the `params` as a parameter.

Reply:
Done.